roboslack-webhook/src/test/java/com/palantir/roboslack/webhook/EnrichTestMessageRequest.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.roboslack.webhook;
+
+import static com.google.common.base.Preconditions.checkArgument;
+
+import com.palantir.roboslack.api.MessageRequest;
+import com.palantir.roboslack.api.attachments.Attachment;
+import java.util.function.BiFunction;
+import org.junit.jupiter.api.TestInfo;
+
+final class EnrichTestMessageRequest implements BiFunction<MessageRequest, TestInfo, MessageRequest> {
```

Comment:
Kinda neat idea we could expand upon if we like... Junit5 provides metadata on the test's themselves, so I figured I would add some to the message that gets sent so we have a visual way to debug it after it was sent.